src/blocks/youtube.v1.yaml

```diff
@@ -17,6 +17,12 @@ properties:
         title: Height
         type: integer
         minimum: 1
+    caption:
+        title: Caption
+        type: array
+        items:
+            oneOf:
+              - $ref: ../blocks/paragraph.v1.yaml
```

Comment:
@thewilkybarkid since we only need the `caption` property, could this be the best solution?